src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -385,6 +387,7 @@ uct_dc_mlx5_iface_dci_release(uct_dc_mlx5_iface_t *iface, uint8_t dci_index)
 
     pool->stack_top--;
     pool->stack[pool->stack_top] = dci_index;
+    iface->tx.dcis[dci_index].ep = NULL;
```

Comment:
remove?

Reply:
not sure it is good idea: uct_dc_mlx5_iface_dci_release is used in several places where dci->ep is not updated

Reply:
which places? it wasn't needed before this PR..

Reply:
dci_put and dci_release_progress

fixed